.grabl/automation.yml

```diff
@@ -214,7 +214,8 @@ release:
       dependencies: [deploy-github]
       command: |
         docker login -u $REPO_DOCKER_USERNAME -p $REPO_DOCKER_PASSWORD
-        bazel run //:deploy-docker
+        bazel run //:deploy-docker-latest
+        bazel run //:deploy-docker-versioned
```

Comment:
There's now 2 targets for deploying containers with different tags. The new `container_push` does not support multi-tag declarations.